src/qt/rpcconsole.cpp

```diff
@@ -415,35 +417,15 @@ void RPCExecutor::request(const QString &command, const WalletModel* wallet_mode
         std::string result;
         std::string executableCommand = command.toStdString() + "\n";
 
-        // Catch the console-only-help command before RPC call is executed and reply with help text as-if a RPC reply.
-        if(executableCommand == "help-console\n") {
-            Q_EMIT reply(RPCConsole::CMD_REPLY, QString(("\n"
-                "This console accepts RPC commands using the standard syntax.\n"
-                "   example:    getblockhash 0\n\n"
-
-                "This console can also accept RPC commands using the parenthesized syntax.\n"
-                "   example:    getblockhash(0)\n\n"
-
-                "Commands may be nested when specified with the parenthesized syntax.\n"
-                "   example:    getblock(getblockhash(0) 1)\n\n"
-
-                "A space or a comma can be used to delimit arguments for either syntax.\n"
-                "   example:    getblockhash 0\n"
-                "               getblockhash,0\n\n"
-
-                "Named results can be queried with a non-quoted key string in brackets using the parenthesized syntax.\n"
-                "   example:    getblock(getblockhash(0) 1)[tx]\n\n"
-
-                "Results without keys can be queried with an integer in brackets using the parenthesized syntax.\n"
-                "   example:    getblock(getblockhash(0),1)[tx][0]\n\n")));
-            return;
-        }
-        if (!RPCConsole::RPCExecuteCommandLine(m_node, result, executableCommand, nullptr, wallet_model)) {
-            Q_EMIT reply(RPCConsole::CMD_ERROR, QString("Parse error: unbalanced ' or \""));
-            return;
+        // Attempt to execute console-only commands
+        if (!RPCExecutor::executeConsoleOnlyCommand(executableCommand, wallet_model)) {
```

Comment:
```suggestion
        if (RPCExecutor::executeConsoleOnlyCommand(executableCommand, wallet_model)) {
            return;
        }
```
(or could put `return` on the same line)